docker-build-images/centos7-build.Dockerfile-ppc64le

```diff
@@ -0,0 +1,22 @@
+FROM ppc64le/centos:7
+MAINTAINER Shaun Crampton <shaun@tigera.io>
+ENV STREAM el7
+
+ARG UID
+ARG GID
+
+# Some commands that would typically be run at container build time must be run in a privileged container.
```

Comment:
I was surprised by this error as well.  I found others complaining about the same issue.  Yes "docker build" runs as root,  however the root user's capabilities may be limited by docker when it starts a container, or in this case the build process.  If my Dockerfile has:
  RUN groupadd --force --gid=1000 user && useradd -o --home=/ --gid=1000 --uid=1000 user

The docker build reports this error, as the process is missing some required capability.
>groupadd: /etc/group.7: Operation not permitted

Unfortunately there is no way to increase the privileged level of a docker build.I suspect this is a bug in docker,  I don't know why the problem is not seen on x86. My workaround should continue to work even if the problem is solved in the future. I will try using the entrypoint approach as you suggested.